src/Tag/Tag.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { Icon } from '../Icon';
+import { Box } from '../Box';
+import { Typography } from '../Typography';
+import { TagWrapper } from './Tag.styles';
+import { getTagIconSize, getTagTypographyVariant } from './utils';
+import { TagProps } from './types';
+
+export const Tag = ({
+  label,
+  backgroundColor = 'purple.main',
+  borderColor = 'purple.700',
+  iconColor = 'white.main',
+  textColor = 'white.main',
+  variant = 'filled',
+  size = 'medium',
+  startIcon,
+  endIcon,
+  onClickEndIcon,
+}: TagProps) => (
+  <TagWrapper $backgroundColor={backgroundColor} $borderColor={borderColor} $variant={variant}>
```

Comment:
A ...rest props parameter should be used by the TagWrapper component in order to allow accessibility and style extension with styled components in case of special cases. This follows one of the standards defined in our design system development specifications.